core/src/main/resources/jenkins/security/apitoken/ApiTokenPropertyConfiguration/help-usageStatisticsEnabled.html

```diff
@@ -0,0 +1,7 @@
+<div>
+    When a user is authenticated by an API Token, its usage counter is incremented and its last usage date updated.
```

Comment:
nit: I would reword this blurb to "If this option is enabled, then the date of the most recent use of each API token and the total number of times it has been used are stored in Jenkins. This allows users to see if they have unused or outdated API tokens which should be revoked."

Maybe we should also add a disclaimer like "This data is stored locally in Jenkins and will not be used for any other purpose" in case someone sees "usage statistics" and gets worried?